store/views.py

```diff
@@ -1,3 +1,58 @@
-from django.shortcuts import render
+import json
+import jwt
+import bcrypt
 
-# Create your views here.
+from .models            import Store, City, District
+
+from django.views       import View
+from django.http        import HttpResponse
+from django.http        import JsonResponse
+from django.db.models   import Q
+
+class StoreMainView(View):
+    def get(self, request):
+        stores = Store.objects.filter(city_id = 9)[:49].values()
+
+        return JsonResponse({'stores':list(stores)}, status = 200)
+    
+
+class CityView(View):
+    def get(self, request):
+        cities = City.objects.values()
+
+        return JsonResponse({'cities':list(cities)}, status = 200)
+
+class DistrictView(View):
+    def get(self, request):
+        city = request.GET.get('city', None)
+        districts = District.objects.filter(city_id = city).values()
+
+        return JsonResponse({'districts':list(districts)}, status = 200)
+
+class StoreSearchView(View):
+    def get(self, request):
+        city      = request.GET.get('city', None)
+        district  = request.GET.get('district', None)
+        store     = request.GET.get('store', None)
+
+        if not city and not district and not store:
+            stores = Store.objects.values() 
+            return JsonResponse({'stores':list(stores)}, status = 200)
+
+        if store:
+            if district:
+                stores = Store.objects.filter(city_id = city, district_id = district, name__icontains = store).values()
```

Comment:
store, district 정보만 입력되면 하기 쿼리를 의도한 것 같은데 확인 부탁 드립니다. city가 None으로 들어가면 데이터가 검색안될것 같아서요.
```suggestion
                stores = Store.objects.filter(district_id = district, name__icontains = store).values()
```